kafka/integration_test.go

```diff
@@ -2376,20 +2386,23 @@ func (its *IntegrationTestSuite) TestProducerConsumerHeaders() {
 
 func TestIntegration(t *testing.T) {
 	its := new(IntegrationTestSuite)
+	its.skipFlaky = true
 	testconfInit()
 	if !testconfRead() {
-		t.Skipf("testconf not provided or not usable")
+		t.Skipf("testconf not provided or not usable\n")
 		return
 	}
-	if testconf.Docker {
-		its.compose = testcontainers.NewLocalDockerCompose([]string{"testresources/docker-compose.yaml"}, "test-docker")
+	fmt.Fprintf(os.Stdout, "TestconfRead was successful!\n")
+	if testconf.Docker && !testconf.Semaphore {
+		its.compose = testcontainers.NewLocalDockerCompose([]string{"./testresources/docker-compose.yaml"}, "test-docker")
 		execErr := its.compose.WithCommand([]string{"up", "-d"}).Invoke()
 		if err := execErr.Error; err != nil {
-			its.T().Fatal(execErr)
+			fmt.Fprintf(os.Stderr, "up -d command failed with the error message %s\n", err)
```

Comment:
Use t.Fatalf and combine this Fprintf  with t.Fatal

Reply:
sure